.github/workflows/publish-to-pypi.yml

```diff
@@ -10,23 +10,10 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: 3.7
-      - name: Install pep517
-        run: >-
-          python -m
-          pip install
-          pep517
-          --user
-      - name: Build a binary wheel and a source tarball
-        run: >-
-          python -m
-          pep517.build
-          --source
-          --binary
-          --out-dir dist/
-          .
-      - name: Publish distribution 📦 to PyPI
-        if: startsWith(github.event.ref, 'refs/tags')
```

Comment:
@marcuxyz I believe we need to keep this line, otherwise we will publish to PyPI every time we merge code on master. We want to publish only when a release is done.

Reply:
Sorry, i forget this.